config/routes.rb

```diff
@@ -12,7 +12,7 @@
 
   # user
   resources :users, only: [:show]
-
+  get 'feed.rss', to: 'feeds#rss', :format => 'rss'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.